pkg/products/marin3r/reconciler.go

```diff
@@ -223,3 +254,148 @@ func (r *Reconciler) preUpgradeBackupExecutor() backup.BackupExecutor {
 	//todo add backup for redis once it's added to the reconciler
 	return backup.NewNoopBackupExecutor()
 }
+
+func (r *Reconciler) reconcilePromStatsdExporter(ctx context.Context, client k8sclient.Client, namespace string) (integreatlyv1alpha1.StatusPhase, error) {
+	logrus.Infof("Start reconcilePromStatsdExporter for marin3r")
+
+	deployment := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "prom-statsd-exporter",
+			Namespace: namespace,
+		},
+	}
+
+	_, err := controllerutil.CreateOrUpdate(ctx, client, deployment, func() error {
+		if deployment.Labels == nil {
+			deployment.Labels = map[string]string{}
+		}
+		deployment.Labels["app"] = "prom-statsd-exporter"
+
+		var replicas int32 = 1
+		deployment.Spec.Replicas = &replicas
+		deployment.Spec.Selector = &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"app": "prom-statsd-exporter",
+			},
+		}
+		deployment.Spec.Strategy = appsv1.DeploymentStrategy{
+			Type: appsv1.RecreateDeploymentStrategyType,
+		}
+		deployment.Spec.Template = corev1.PodTemplateSpec{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: map[string]string{
+					"app": "prom-statsd-exporter",
+				},
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:  "prom-statsd-exporter",
+						Image: "prom/statsd-exporter:v0.18.0",
+						Ports: []corev1.ContainerPort{
+							{
+								Name:          "prom-statsd",
+								ContainerPort: 9125,
+							},
+							{
+								Name:          "metrics",
+								ContainerPort: 9102,
+							},
+						},
+					},
+				},
+			},
+		}
+
+		return nil
+	})
+
+	if err != nil {
+		return integreatlyv1alpha1.PhaseFailed, err
+	}
+
+	return integreatlyv1alpha1.PhaseCompleted, nil
+}
+
+func (r *Reconciler) reconcilePromStatsdExporterService(ctx context.Context, client k8sclient.Client, namespace string) (integreatlyv1alpha1.StatusPhase, error) {
+	logrus.Infof("Start reconcilePromStatsdExporterService for marin3r")
+
+	service := &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "prom-statsd-exporter",
```

Comment:
What do you think on using `statsdHost` constant here and other places that use `prom-statsd-exporter` ? 
Might be unlikely this will change but would be good to have one place to change the name since this is referenced quite a bit as a label `selector` :thinking:

Reply:
Yeah, it's cleaner